### DISCUSSION TITLE
Org name is changing

### DISCUSSION THREAD

**Discussion by erlend-sh** [fishfight/punchy#197] [Planning]
I’m about to change our org name to reflect our newest and likely final branding. Are there any loose ends to tie up before/after I do this?

It’s already late here so if I don’t hear back (which would be fine!) within an hour I’ll hold off on the rename until tomorrow.

**Comment by zicklag** [comment]
I don't know of anything we need to do before.

After we'll need to update our links, but I think that's it.

**Reply by erlend-sh** [reply]
Changed! Following up with itch announcement tomorrow.

**Reply by zicklag** [reply]
Sweet. Update links in #199.
